ffi/wasm/bench/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Suggestion: please add a `npm` task here so that one can run something like `npm run bench`.